### PR TITLE
Added pyconindia 2016 CFP blog

### PR DESCRIPTION
**Pull request by surajnarwade**

### CODE REVIEW

File: content/blog/cfp-open-for-pycon-india-2016.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Title: CFP are open NOW!
```
Comment (akshayaurora) [inline content/blog/cfp-open-for-pycon-india-2016.md]: Have been open, they don't open now.

File: content/blog/cfp-open-for-pycon-india-2016.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Date: 2016-06-08 14:07
+Category: 2016
+Slug: 2016-CFP-open
+Summary: PyCon India 2016 - CFP link and details.
+
+CFP for [PyCon India](http://in.pycon.org/2016) is open now!
```
Comment (akshayaurora) [inline content/blog/cfp-open-for-pycon-india-2016.md]: Same as above

File: content/blog/cfp-open-for-pycon-india-2016.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+[Submit Now!](https://in.pycon.org/cfp/2016/proposals/) Read the guidelines properly,
+and submit the proposals now.
+
+PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community. It attracts the best Python programmers across the country and abroad.
+CFP schedule is as follows:
```
Comment (akshayaurora) [inline content/blog/cfp-open-for-pycon-india-2016.md]: We need better write up, this is just taken up from the blog

**Comment by vnbang2003** [comment]
@kracekumar Can you please review and accept this merge request of everything is fine.

**Comment by kracekumar** [comment]
@vnbang2003 Author hasn't fixed the comments. @surajnarwade Can you fix it or if not let us know ?

**Comment by kracekumar** [comment]
I am rejecting the PR in favour of https://github.com/pythonindia/inpycon-blog/pull/38

**Pull request merged**